quality/README.md

```diff
@@ -6,6 +6,11 @@ It resolves the following
 * https://w3id.org/quality/qskos
 * https://w3id.org/quality/voc/wcol
 * https://w3id.org/quality/voc/pref
+* http://w3id.org/quality/exp/s170905/
+* http://w3id.org/quality/exp/170905
+* http://w3id.org/quality/exp/AHPOnSKOSTemplate
+* http://w3id.org/quality/exp/preferences
+* http://w3id.org/quality/DQV/implementations
```

Comment:
These should be https since w3id.org will redirect http to https.

Reply:
Thanks, I have updated the file,  it is now Ok?